code/game/objects/effects/landmarks.dm

```diff
@@ -443,6 +443,67 @@ INITIALIZE_IMMEDIATE(/obj/effect/landmark/start/new_player)
 	icon_state = "portal_exit"
 	var/id
 
+/**
+ * Landmark to designate certain areas that do not need to exist nor be loaded at world start
+ * but do want to be loaded under certain circumstances. Use this for stuff like the nukie base or wizden, aka stuff that only matters when their antag is rolled.
+ */
+/obj/effect/landmark/lazy_template_pivot
+	name = "lazy template pivot"
+	icon_state = "lazy_pivot" // todo: icons for all this shit
+	var/key
+	var/map_path
+
+/obj/effect/landmark/lazy_template_pivot/Initialize(mapload)
+	GLOB.lazy_template_pivots += src
+	return ..()
+
+/obj/effect/landmark/lazy_template_pivot/Destroy(...)
+	GLOB.lazy_template_pivots -= src
+	return ..()
+
+/**
+ * Does the grunt work of loading the template onto itself.
+ * Has a basic check to ensure that the loaded template isn't larger than the preallocated area.
+ */
+/obj/effect/landmark/lazy_template_pivot/proc/lazy_load()
+	var/turf/my_turf = get_turf(src)
+
+	if(!map_path || !fexists(map_path))
+		CRASH("lazy_template_pivot [type] has an invalid map_path: '[map_path]'")
+
+	var/datum/map_template/loading = new(path = map_path, cache = TRUE)
+
+	// ensure that what we're loading isn't larger than the allocation
+	var/list/affected = loading.get_affected_turfs(my_turf)
+	for(var/turf/turf as anything in affected)
+		var/area/turf_area = get_area(turf)
+		if(turf_area.type != /area/misc/lazy_pivot_allocation)
+			CRASH("Attempted to load a lazy template pivot larger than the allocation for the pivot")
```

Comment:
could we acomplish this with a variable on the landmark? loading non default space turfs is slower then just default ones

Reply:
not without losing a sanity guard check against mappers increasing the size of a template without updating the pivot

Reply:
compare the size of the map to the size defined on the thing. a feature of the maploader is being able to dryload map files to get an idea of their size before placing them in the world